src/Bicep.Core/TypeSystem/TypeAssignmentVisitor.cs

```diff
@@ -687,6 +749,11 @@ public override void VisitInstanceFunctionCallSyntax(InstanceFunctionCallSyntax
 
         public override void VisitTargetScopeSyntax(TargetScopeSyntax syntax)
             => AssignTypeWithDiagnostics(syntax, diagnostics => {

```

Comment:
I just realized that this never got called and type checking is not working for `TargetScopeSyntax` (same on the main branch). Setting `targetScope` to some random value won't produce any diagnostics:
![image](https://user-images.githubusercontent.com/16367959/105431405-65116800-5c0a-11eb-9dd3-5d42dab19123.png)

This seems to me a regression because it used to work in 0.1.223-alpha. I'm not sure which commit break it though.

Reply:
Thanks - good find!